### DISCUSSION TITLE
Add Qt QML/JS engine

### DISCUSSION THREAD

**Discussion by ratijas** [mdn/mdn-community#401] [Browser compatibility data]
Qt is a cross-platform application and UI framework. It comes with its own language for declarative markup called QML, which is based on their custom, fairly specific but powerful JS engine called QV4.

After some prolonged period of slow-paced development during Qt 5.x times, recent Qt 6.x releases are rapidly catching up with more and more modern ECMAScript features getting proper support. Meanwhile in KDE/Plasma we still have to maintain LTS branch based on 5.x version, and we'll be doing so for quite some time yet.

Thus, it often would be very useful to have compatibility data available in centralized place at MDN, which is by far the most valuable resource to look for JavaScript documentation.

Maintaining such data would be a huge task though, and I can't even imagine where to start. I seek help and advice.

---

https://doc.qt.io/qt-6/qtqml-index.html

**Comment by pepelsbey** [comment]
Thank you for starting the discussion. I’m closing this now due to inactivity, but if you think that’s a mistake, feel free to reply. I’d also suggest filing an issue directly in the [browser-compat-data repository](https://github.com/mdn/browser-compat-data/issues), which MDN uses for showing the browser support tables. You’re also welcome to join the [MDN Discord](https://developer.mozilla.org/discord) if you’d prefer to chat with us more about it in sync.
